start-android.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+mkdir -p android/app/src/main/assets
```

Comment:
just curious, how come this needs to be done?

Reply:
Answered in previous comment.